packages/ketcher-react/src/script/render/restruct/reatom.js

```diff
@@ -250,6 +260,42 @@ class ReAtom extends ReObject {
   }
 }
 
+function getStereoAtomColor(options, stereoLabel) {
+  if (
+    !stereoLabel ||
+    options.colorStereogenicCenters === StereoColoringType.Off ||
+    options.colorStereogenicCenters === StereoColoringType.BondsOnly
+  ) {
+    return '#000'
+  }
+
+  return getColorFromStereoLabel(options, stereoLabel)
+}
+
+export function getColorFromStereoLabel(options, stereoLabel) {
+  const stereoLabelType = stereoLabel.match(/\D+/g)[0]
+
+  switch (stereoLabelType) {
+    case StereoLabel.And:
+      return options.colorOfAndCenters
+    case StereoLabel.Or:
+      return options.colorOfOrCenters
+    case StereoLabel.Abs:
+      return options.colorOfAbsoluteCenters
+    default:
+      return '#000'
+  }
+}
+
+function getStereoAtomOpacity(stereoLabel) {
+  const stereoLabelType = stereoLabel.match(/\D+/g)[0]
+  const stereoLabelNumber = +stereoLabel.replace(stereoLabelType, '')
+  if (stereoLabelType === StereoLabel.Abs) {
+    return 1
+  }
+  return Math.max(1 - (stereoLabelNumber - 1) / 10, 0.3)
+}
+
 function isLabelVisible(restruct, options, atom) {
   const visibleTerminal =
     options.showHydrogenLabels !== 'off' &&
```

Comment:
Please Use enum type instead of 'magic' strings. The file should be transformed to TS